beets/ui/commands.py

```diff
@@ -1364,6 +1359,21 @@ def modify_items(lib, mods, dels, query, write, move, album, confirm):
             obj.try_sync(write, move)
 
 
+def print_modify_item(obj, mods, dels):
```

Comment:
Well, it does call `obj.update()`, which actually modifies the object—so it's not *just* printing. Maybe `update_and_print`?